codegen/protocol-tests/model/error-correction-tests.smithy

```diff
@@ -60,7 +64,7 @@ structure TestStruct {
     @required
     nestedListValue: NestedList
 
-    @required
+    // @required
```

Comment:
Were these `@required` removed to fix some protocol tests? I'd rather leave them disabled than change our test models

Reply:
The required trait isn't supposed to be there, a smithy validator throw an error if it is, even if I disable the protocol test